cmd/server-main.go

```diff
@@ -276,6 +259,99 @@ func isDistributedSetup(eps []*url.URL) (isDist bool) {
 	return isDist
 }
 
+// We just exit for invalid endpoints.
+func checkEndpointsSyntax(eps []*url.URL, disks []string) {
+	for i, u := range eps {
+		// Scheme is "" for FS and singlenode-XL
+		if u.Scheme != "" && u.Scheme != "http" && u.Scheme != "https" {
```

Comment:
This not correct on windows when you specify C:\disk windows keeps "C" as part of Scheme.  You can possibly validate this on windows in this manner.  You have to handle this just like how getPath() handles the right path from looking at the list of URLs on windows.